internal/wstrust/wstrust_mex_document.go

```diff
@@ -49,9 +51,14 @@ func updateEndpoint(cached *Endpoint, found Endpoint) bool {
 
 // TODO(jdoak): Refactor into smaller bits
 
-func CreateWsTrustMexDocument(responseData string) (MexDocument, error) {
+func CreateWsTrustMexDocument(resp *http.Response) (MexDocument, error) {
 	definitions := &definitions{}
```

Comment:
Nit: I'd move definitions down to right above line 61 where it is first used.

Reply:
Fixed.